cloudify_rest/rest_sdk/utility.py

```diff
@@ -81,13 +83,24 @@ def _send_request(call):
                                         data=data,
                                         json=json_payload,
                                         verify=call['verify'])
-        except requests.exceptions.ConnectionError:
+        except requests.exceptions.ConnectionError as e:
             logger.debug('ConnectionError for host : {}'.format(host))
+
+            if TEMPLATE_PROPERTY_RETRY_ON_CONNECTION_ERROR in call and \
+                    call[TEMPLATE_PROPERTY_RETRY_ON_CONNECTION_ERROR]:
+
+                raise RecoverableResponseException(
+                    'ConnectionError {0} has occurred, but flag {1} is set. '
+                    'Retrying...'
+                    .format(
+                        str(e),
+                        TEMPLATE_PROPERTY_RETRY_ON_CONNECTION_ERROR
+                    )
+                )
+
             if i == len(call['hosts']) - 1:
```

Comment:
Why change from error to warn?

Reply:
It was misleading before but now without now flag set it should be in fact 'error'. Corrected.